2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -214,13 +215,149 @@ Viene inserito ";" alla fine delle voci dell'elenco tranne l'ultima che termina
+Diventa compito del responsabile stabilire le attività necessarie per attuare le contromisure, e di conseguenza di pianificare le attività da svolgere nello sprint successivo. \
+Il risultato di questo processo è visibile all'interno del Piano di Progetto.
+\
+\
+==== Ruoli 
+===== Responsabile
+Il responsabile è la figura chiave che guida il progetto, rappresentandolo nei confronti degli stakeholders. Le sue responsabilità sono molteplici:
+
++ coordinamento: ha il compito di supervisionare i membri del gruppo, assicurandosi che le attività siano svolte in modo efficiente e tempestivo;
+
++ pianificazione: il responsabile è incaricato di delineare il percorso del progetto, stabilendo le attività da svolgere, le relative scadenze e priorità, sancendo l'inizio e la fine di ogni sprint;
+
++ monitoraggio e gestione dei costi: il responsabile tiene sotto controllo l'andamento del progetto, stima i costi e gestisce l'analisi dei rischi, garantendo che il progetto rimanga entro il budget previsto;
+
++ norme di progetto: il responsabile è responsabile della stesura e dell'aggiornamento delle norme di progetto, che devono essere rispettate da tutti i membri del gruppo;
+
++ relazioni esterne: gestisce tutte le interazioni con la proponente e i committenti assicurando una comunicazione fluida e efficace;
+
+In sintesi, il responsabile è il fulcro che mantiene il gruppo unito e guida il progetto verso il successo.
+\ \
+===== Amministratore
+L'amministratore è la figura che gestisce l'ambiente di lavoro e gli strumenti utilizzati dal gruppo per tutta la durata del progetto. Ha il compito di assicurare che gli strumenti proposti ai membri del gruppo siano efficienti e favoriscano la qualità del lavoro. Monitora il rispetto delle regole stabilite in questo documento e verifica che i servizi a disposizione del team siano adeguati alle attività pianificate, promuovendo la produttività. \
+
+Le responsabilità dell'amministratore includono:
+
++ redazione dei verbali: l'amministratore è responsabile della stesura dei verbali relativi ai meeting interni ed esterni;
+
++ gestione delle risorse: si occupa dell'amministrazione delle risorse, delle infrastrutture e dei servizi necessari per l'esecuzione dei processi di supporto;
+
++ automatizzazione dei processi: determina gli strumenti necessari per automatizzare i processi, come la compilazione dei file sorgenti e il sistema di versionamento automatico;
+
++ risoluzione dei problemi: affronta e risolve i problemi legati alla gestione dei processi.
+
+In sintesi, l'amministratore è il motore che mantiene il progetto in movimento, assicurando che tutto funzioni senza intoppi.
+
+\ \
+===== Analista
+L'analista svolge un ruolo estremamente delicato, facendosi carico di individuare i needs del proponente e di trasformarli in requisiti che saranno l'input delle attività successive. Il suo lavoro si svolge intensamente nel periodo di avvio del progetto, e si conclude con la stesura dell'Analisi dei Requisiti. \
```

Comment:
non sono sicuro serva usare "needs" (bisogni va bene lo stesso credo) in inglese almeno che non si tratti di un termine tecnico, nel caso la pensi diversamente lascio a te la scelta comunque.